src/commons/utils/corio_utils.py

```diff
@@ -265,6 +275,39 @@ def setup_environment():
     os.makedirs(DATA_DIR_PATH, exist_ok=True)
 
 
+async def activate_degraded_mode() -> None:
+    """Activate degraded mode for execution by shutdown/deleting pod."""
+    bucket = f'object-op-{time.perf_counter_ns()}'.lower()
+    access_key = literal_eval(os.getenv("options")).get("access_key")
+    secret_key = literal_eval(os.getenv("options")).get("secret_key")
+    endpoint_url = literal_eval(os.getenv("options")).get("endpoint_url")
+    use_ssl = literal_eval(os.environ.get("options")).get("use_ssl")
+    response = await S3Bucket(access_key, secret_key, endpoint_url=endpoint_url,
+                              use_ssl=use_ssl).create_bucket(bucket)
+    if response:
+        os.environ["d_bucket"] = bucket
```

Comment:
Will this same bucket be used for all the operations/tests cases

Reply:
Yes, this same bucket will be used for all object operations as only object operation suite is supported for degraded mode.